frontend/svelte.config.js

```diff
@@ -15,7 +15,9 @@ const config = {
 		alias: {
 			$comp: './src/components',
 			$lib: './src/lib',
-			$params: './src/params'
+			$params: './src/params',
+			$types: './src/types',
```

Comment:
This is fine, but `$types` might be confusing with the auto-generated `./$types` already existing